uqcsbot/haiku.py

```diff
@@ -43,6 +44,11 @@ async def on_message(self, message):
         else:
             await message.reply(f"Nice haiku:\n{haiku}")
 
+    @app_commands.command()
+    @app_commands.describe(text="Syllable check for a given word")
```

Comment:
This should refer to the word arg, instead of text

```suggestion
    @app_commands.describe(word="Syllable check for a given word")
```